tests/test_pytest_plugin.py

```diff
@@ -68,7 +83,8 @@ async def test_skip_inline(some_feature):
     )
 
     result = testdir.runpytest('-v')
-    result.assert_outcomes(passed=3 * len(get_all_backends()) + 1, skipped=len(get_all_backends()))
+    result.assert_outcomes(passed=3 * len(get_all_backends()) + 1, skipped=len(get_all_backends()),
+                           failed=1)
```

Comment:
I'd expect test_asyncio_exception_handler to pass, then fail in teardown:

```suggestion
    result.assert_outcomes(passed=3 * len(get_all_backends()) + 2, skipped=len(get_all_backends()),
                           failed=1)
```

Reply:
Why? The exception is raised during the actual test run (albeit in a callback), not during fixture teardown.